exercises/pangram/example.js

```diff
@@ -1,10 +1,10 @@
-const notAlpha = /[^a-z]+/gi,
-  alphaLength = 26;
-let cleaned,
-  sortedSet;
+const notAlpha = /[^a-z]+/gi;
+const alphaLength = 26;
+let cleaned;
+let sortedSet;
```

Comment:
Not to split hairs, but is there is any reason `cleaned` and `sortedSet` need to be defined here at all? Seems like it would be better to just declare them as `const`s where they are used, like:

```js
export const isPangram = (candidate) => {
  const cleaned = (candidate.replace(notAlpha, '')).toLowerCase();
  const sortedSet = new Set([...cleaned].sort());
  return sortedSet.size === alphaLength;
};
```

Reply:
@gargrave you're right it is not needed to define them there, updated the PR accordingly 👍